### PR TITLE
Update dockerfile for scancontrol SDK version 0.2.5

### PR DESCRIPTION
**Pull request by dave992**
The scanCONTROL SDK has been changed from v0.2.4 to v0.2.5. Due to the different naming, the docker image build is failing and needs to be updated. These changes are aimed at using the new version and names in the Dockerbuild file.

### CODE REVIEW

File: .docker/Dockerfile [hunk 1]
```diff
@@ -27,25 +27,25 @@ RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz
     checkinstall --pkgname aravis --requires="libglib2.0-dev, libxml2-dev" && \
     ldconfig
 
-RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-0-2-4.zip && \
-    unzip scanCONTROL-Linux-SDK-0-2-4.zip && \
-    rm scanCONTROL-Linux-SDK-0-2-4.zip && \
-    cd /scanCONTROL\ Linux\ SDK\ 0.2.4/libmescan/ && \
+RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-0-2-5.zip && \
```
Comment (gavanderhoorn) [inline .docker/Dockerfile:30]: You could use an `ARG` and set it to a default value (ie: the version you're targetting) to keep this DRY.

Comment (dave992) [reply .docker/Dockerfile]: Done :)

File: .docker/Dockerfile [hunk 2]
```diff
@@ -59,5 +64,5 @@ COPY --from=build ["/library_pkgs", "/library_pkgs"]
 
 RUN apt-get update && \
     apt install -y /library_pkgs/aravis_0.6.4-1_amd64.deb && \
-    apt install /library_pkgs/mescan_0.2.4-1_amd64.deb && \
-    apt install /library_pkgs/llt_0.2.4-1_amd64.deb
\ No newline at end of file
+    apt install /library_pkgs/mescan_${SCANCONTROL_SDK_VERSION}-1_amd64.deb && \
+    apt install /library_pkgs/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb
```
Comment (gavanderhoorn) [inline .docker/Dockerfile:68]:
```suggestion
    apt install /library_pkgs/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb

```

Comment (dave992) [reply .docker/Dockerfile:68]: Add a white line at the end of the file? I do not see any changes otherwise ;)

Comment (gavanderhoorn) [reply .docker/Dockerfile:68]: Yes. You're file is missing an empty last line.

Github even tells you so ;)

Comment (dave992) [reply .docker/Dockerfile:68]: If that is true, I am just curious about the why behind this

**Comment by dave992** [comment]
I manually pushed the new docker image to the docker hub and reran the CI jobs. The build test is working as intended again.

File: .docker/Dockerfile (hunk 2 above)
Comment (dave992) [reply .docker/Dockerfile]: Ah, I see the warning now indeed. Did not know this was a thing

Comment (gavanderhoorn) [reply .docker/Dockerfile]: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Pull request merged**